src/plugins/validation/oas3/semantic-validators/responses.js

```diff
@@ -52,6 +52,13 @@ module.exports.validate = function({ resolvedSpec }, config) {
       } else {
         // default warnings for discouraged status code per IBM API Handbook
```

Comment:
Okay I have to be a little obnoxious here but for the sake of clarity, this comment should be moved to be just before `if (statusCode === '422') {`. Those are the checks the comment describes, so it is a little confusing to leave here as if it applies to everything.